### ISSUE TITLE
Guidance on fine-tuning V-JEPA 2

### ISSUE THREAD

**Issue by baraql** [facebookresearch/vjepa2#93]
Hello,

I’m fine-tuning the pretrained V-JEPA 2 encoder/predictor on the nuScenes autonomous driving dataset and would appreciate guidance on the hyperparameters. 

Dataset: nuScenes contains ~6,000 camera videos, each about 20 seconds long at 12 Hz, totaling ~1.4M frames. For training, I’m sampling 64-frame clips at 4 Hz from these videos.

Questions:
- Should I just use the pretrain.yaml or cooldown.yaml config in /configs/train/?
- If I should adapt a config, are there suggestions on choosing training steps and a learning rate schedule given a dataset size?
- Are there other hyperparameters you recommend changing when fine tuning?

Thanks!

**Comment by mmuckley** [comment]
Hello @baraql, sorry for the late reply.

I think you probably want to start with the cooldown config.

The main thing you'll need to adjust are EMA and learning rate. JEPA models are very sensitive to EMA in particular, and you may need to try the full range between something like 0.99 and 0.9995. Keep in mind also a difference of like 0.995 vs. 0.996 can be important for performance, depending on the application, but for video we usually stay above 0.999.

Good luck!

**Comment by technicallydubious** [comment]
@mmuckley could you please provide more details regarding domain adaptation? 

1. Number of frames: would 16 be sufficient or need to continue training on 64 frames 
2. Data size: how much data would be considered sufficient for domain adaptation? 
3. Is linear probing for some task the best way to test the domain adaptation, i.e. motion prediction on driving data?
4. The loss curve goes back up after a few epochs, is this expected?
